content/events/2020-amsterdam/sponsor-form.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Sponsor Form
+---
+**Kubernetes Days Amsterdam* 
+
+**September 10-11, 2020** 
+
+**Amsterdam, The Netherlands**
+
+# Sponsorship Application Form
+
+## Host: Cost of Venue (Limit 1)
```

Comment:
I think `Venue Sponsor` is the most common name.

Reply:
fixed